dev/NavigationView/NavigationView.idl

```diff
@@ -270,15 +284,29 @@ unsealed runtimeclass NavigationView : Windows.UI.Xaml.Controls.ContentControl
     Boolean IsTitleBarAutoPaddingEnabled { get; set; };
 
     static Windows.UI.Xaml.DependencyProperty IsTitleBarAutoPaddingEnabledProperty { get; };
+
+    [WUXC_VERSION_PREVIEW]
+    {
+        event Windows.Foundation.TypedEventHandler<NavigationView, NavigationViewExpandingEventArgs> Expanding;
+        event Windows.Foundation.TypedEventHandler<NavigationView, NavigationViewCollapsedEventArgs> Collapsed;
+
+        void Expand(NavigationViewItem item);
+        void Collapse(NavigationViewItem item);
+    }
 }
 
 [WUXC_VERSION_RS3]
 [webhosthidden]
 [WUXC_INTERFACE_NAME("INavigationViewItemBase", edf04eb1-37d1-471f-8570-3829ee5b2bc6)]
 [WUXC_CONSTRUCTOR_NAME("INavigationViewItemBaseFactory", eb014cef-7890-4ebb-8245-02e8510f321d)]
 [default_interface]
-unsealed runtimeclass NavigationViewItemBase : Windows.UI.Xaml.Controls.ListViewItem
+unsealed runtimeclass NavigationViewItemBase : Windows.UI.Xaml.Controls.ContentControl
 {
+    [WUXC_VERSION_PREVIEW]
+    {
+        Boolean IsSelected{ get; set; };
```

Comment:
>IsSelected [](start = 16, length = 10)

NavigationViewItemSeperator is getting a selected property?

Reply:
Yeah, it already had one from ListViewItem. This seems like the most apt choice in case we have other selectable items in the future that derive from NVIB.

---
In reply to: [386534175](https://github.com/microsoft/microsoft-ui-xaml/pull/2004#discussion_r386534175) [](ancestors = 386534175)

Reply:
Or would it be better to have NVIB derive from SelectorItem?

Reply:
I would not recommend extending from SelectorItem for just this property. SelectorItem is designed to work with Selector which we are not using in this case.

Reply:
This is already the case since IsSelected is coming from SelectorItem.

---
In reply to: [387233681](https://github.com/microsoft/microsoft-ui-xaml/pull/2004#discussion_r387233681) [](ancestors = 387233681,386534175)